src/Sarif.UnitTests/TestData/SarifVersionOneToCurrentVisitor/ExpectedOutputs/OneRunWithAllReportingDescriptors.sarif

```diff
@@ -0,0 +1,63 @@
+{
+  "$schema": "http://json.schemastore.org/sarif-2.0.0-csd.2.beta.2019-02-20",
+  "version": "2.0.0-csd.2.beta.2019-02-20",
+  "runs": [
+    {
+      "tool": {
+        "driver": {
+          "name": "CodeScanner",
+          "semanticVersion": "2.1.0",
+          "ruleDescriptors": [
+            {
+              "id": "C2003",
+              "name": "Rule C2003",
+              "shortDescription": {
+                "text": "Rules were meant to be broken."
+              },
+              "fullDescription": {
+                "text": "This is a test for validating modification of tool.driver.ruleDescriptors"
+              },
+              "defaultConfiguration": {
+                "enabled": false,
+                "level": "note",
+                "rank": -1.0
```

Comment:
this is wrong, this is a default value. Did you lose a custom edit to a 'NotYetGeneratedFiles'? Check the default value of the rank attribute, which s/be expressed as -1.0 (not 1.0).

Underlying issue is that if you emit -1 as a default value, this looks like an integer, so the actual default of -1.0 (which is a different JSON numeric type) will be explicit in code.